package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "mattermost-desktop",
+  "desktopName": "Mattermost.desktop",
```

Comment:
desktopName is the name of the .desktop file, used to identify he application in the unity dash, launcher and whenever you use a unity API.

The existence is described here: http://electron.atom.io/docs/all/#winsetprogressbarprogress
& http://electron.atom.io/docs/all/#appsetbadgecountcount-linux-macos

I'm not sure if it is needed in `src/package.json` (I'm not sure how this file is related to `/package.json`).